src/lib/utils/balancer/web3.ts

```diff
@@ -42,10 +42,23 @@ export async function sendTransaction(
     const gasLimit = gasLimitNumber.toNumber();
     overrides.gasLimit = Math.floor(gasLimit * (1 + GAS_LIMIT_BUFFER));
 
-    if (USE_BLOCKNATIVE_GAS_PLATFORM && overrides.gasPrice == null) {
+    if (
+      USE_BLOCKNATIVE_GAS_PLATFORM &&
+      overrides.gasPrice == null &&
+      overrides.maxFeePerGas == null &&
+      overrides.maxPriorityFeePerGas == null
```

Comment:
if any of these are set I think the intent is for manual control over the gas price.

Reply:
If any are changed then shouldn't we use an OR condition

Reply:
I mean, I need all of them to be `null` to proceed (or more likely `undefined`) - but that will match both anyway.